docker/ubuntu/ursa-fossa.dockerfile

```diff
@@ -41,5 +42,5 @@ RUN cd /usr/lib/x86_64-linux-gnu \
     && make install \
     && cd .. \
     && rm -rf libsodium-1.0.18 \
-    && curl https://sh.rustup.rs -sSf | sh -s -- -y
+    && curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly-2020-04-12
```

Comment:
what requires nightly and do you know how far away those things are from being supported on stable?

Reply:
We test for using SIMD instructions with some of the hashes which only work with nightly.